diff --git a/meson.build b/meson.build
index c1f40f3..c80fb9d 100644
--- a/meson.build
+++ b/meson.build
@@ -141,7 +141,7 @@ python3_available_modules = []
 
 foreach module, required : python3_test_modules
     if required and run_command(python3, '-c', 'import @0@'.format(module)).returncode() != 0
-        error('Python3 module \'' + module + '\' required by test suite not found')
+        warning('Python3 module \'' + module + '\' required by test suite not found')
     endif
 endforeach
 
